src/modules/auth/auth.controller.ts

```diff
@@ -33,8 +39,13 @@ export class AuthController {
     status: 401,
     description: 'Invalid email or password',
   })
-  login(@Body() loginDto: LoginDto) {
-    return this.authService.login(loginDto);
+  async login(@Body() loginDto: LoginDto) {
```

Comment:
Please look at the services as well. Some services, like the login service, are returning message with the data. Since we moved message inside baseResponse, service should only return data.